api/routers/areas.py

```diff
@@ -113,16 +119,65 @@ async def create_area(new_area: AreaConfigDTO, reboot_processor: Optional[bool]
     return next((area for area in get_areas() if area["id"] == area_dict["Id"]), None)
 
 
+def modify_area_all(area_information):
+    """
+    Edits the configuration related to the area "ALL", an area that contains all cameras.
+    """
+    # Doubt: We force the user to send "cameras", since the input must be AreaConfig DTO. However, I don't think
+    # this is entirely correct.
+
+    config = get_config()
+    config_directory = config_utils.get_area_config_directory(config)
+    config_path = os.path.join(config_directory, ALL_AREAS + ".json")
```

Comment:
You should use `config.get_area_config_path(ALL_AREAS)`

Reply:
Done!

Reply:
Done!